pkg/sql/compile/compile.go

```diff
@@ -498,6 +512,41 @@ func (c *Compile) compileExternScan(n *plan.Node) []*Scope {
 	return ss
 }
 
+func (c *Compile) compileUnnest(n *plan.Node, pre []*Scope) ([]*Scope, error) {
+	ds := &Scope{}
+	args := &tree.UnnestParam{}
+	err := args.Unmarshal(n.TableDef.TableFunctionParam)
+	if err != nil {
+		return nil, err
+	}
+	if args.IsCol {
```

Comment:
What does isCol mean? I am not very clear, can you tell me?

Reply:
It means whether the datasource is a column in table. If not, the source is a str from cmd line.

Reply:
ok